mne/__init__.py

```diff
@@ -48,9 +48,8 @@
                       make_forward_dipole)
 from .source_estimate import (read_source_estimate, MixedSourceEstimate,
                               SourceEstimate, VectorSourceEstimate,
-                              VolSourceEstimate, morph_data,
-                              morph_data_precomputed, compute_morph_matrix,
-                              grade_to_tris, grade_to_vertices,
```

Comment:
@TommyClausner by doing this you break the code of many people without warning.

I'll revert this so users just have a deprecation warning and not a crash.